dashboard/config/foorm/forms/surveys/teachers/teacher_end_of_year_survey.0.json

```diff
@@ -589,15 +680,15 @@
     },
     {
      "type": "comment",
-     "name": "one_challenge_FR_normal",
-     "title": "For one of the items you identified as a huge challenge, please describe any ideas you have for how code.org might address it.",
+     "name": "one_challenge_FR_remote",
+     "title": "For one of the items you identified as a huge challenge (or for a different challenge not listed), please describe any ideas you have for how code.org might address it.",
```

Comment:
do you still want huge challenge here since it was removed as an option?

Reply:
Also, "code.org" should be "Code.org".